storage/innobase/ut/ut0crc32.cc

```diff
@@ -733,27 +730,17 @@ ut_crc32_init()
 	probably kill your program.
 
 	*/
-#ifndef UNIV_DEBUG_VALGRIND
-	ut_crc32_sse2_enabled = (features_ecx >> 20) & 1;
-#endif /* UNIV_DEBUG_VALGRIND */
 
-	if (ut_crc32_sse2_enabled) {
+	if ((features_ecx >> 20) & 1) {
```

Comment:
I think that `features_ecx & 1 << 20` would translate into a single instruction, while this one may require 2 unless a compiler optimization is rewriting it to what I suggested.